test/trento/application/projectors/host_projector_test.exs

```diff
@@ -148,8 +148,12 @@ defmodule Trento.HostProjectorTest do
     }
 
     ProjectorTestHelper.project(HostProjector, event, "host_projector")
-    projection = Repo.get!(HostReadModel, host_id)
 
+    assert_broadcast "host_details_updated",
```

Comment:
Why did you change the order between the `assert_broadcast` and `Repo.get`?
Every other test has the `assert_broadcast` as last thing in the test